solutions/models.py

```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django_nanoid.models import NanoIDField 
+from utils.choices import Sender
+
+# Create your models here.
+
+class SolutionHistory(models.Model):
+    id = NanoIDField(
+    primary_key=True,
+    editable=False,
+    unique=True,
+    max_length=21,
+    )
+    user=models.ForeignKey('accounts.User', on_delete=models.CASCADE, related_name='solutionhistory')
+    mission=models.ForeignKey('missions.Mission', on_delete=models.CASCADE, related_name='solutionhistory')
```

Comment:
❗ Code Convention에서 변수명은 snake_case를 사용하기로 했었습니다. https://www.notion.so/Code-Convention-276cc780cd7d806ea542fe3b9748257d#276cc780cd7d80b3976ed49e18ab9a51
```suggestion
    user=models.ForeignKey('accounts.User', on_delete=models.CASCADE, related_name='solution_history')
    mission=models.ForeignKey('missions.Mission', on_delete=models.CASCADE, related_name='solution_history')
```